Bidding/in-market-audiences-bidding.js

```diff
@@ -235,8 +239,12 @@ function campaignHasAnyCampaignLevelAudiences(campaign) {
   return totalNumEntities > 0;
 }
 
-function applyBids(operations) {
+function applyBids(operations, audienceMapping) {
   operations.forEach(function (operation) {
+    message = " - Updating " + operation.entityType + ": '" + operation.entityName + "'; "
```

Comment:
I'm a bit confused as to what you mean about blank space/new lines?

Reply:
This will print:
` - Updating Campaign: 'BL|Woo'; Audience: 'This audience'; New Modifier: 0.4`
` - Updating Campaign: 'BL|two'; Audience: 'That audience'; New Modifier: 0.5`